src/vm/appdomain.cpp

```diff
@@ -6389,6 +6392,9 @@ HRESULT RuntimeInvokeHostAssemblyResolver(INT_PTR pManagedAssemblyLoadContextToB
             // Step 3 (of CLRPrivBinderAssemblyLoadContext::BindUsingAssemblyName)
             if (!fResolvedAssembly && !isSatelliteAssemblyRequest)
             {
+                tracer.Trace(BinderTracing::ResolutionAttemptedOperation::Stage::PlatformAssemblies,
```

Comment:
`PlatformAssemblies` > `DefaultALCFallback`, since we are starting the `DefaultALCFallback` stage
(Or maybe I'm not understanding how the `ResolutionAttemptedOperation` class is being used)

Reply:
Isn't step 3 the look up within the TPA Binder?